spec/ddtrace/integration_spec.rb

```diff
@@ -493,8 +493,8 @@ def agent_receives_span_step3(previous_success)
   describe 'tracer transport' do
     subject(:configure) do
       Datadog::Tracing.configure do |c|
-        c.tracer.hostname = hostname
-        c.tracer.port = port
+        c.agent.host = hostname
+        c.agent.port = port
```

Comment:
This seems like it will/should fail, since it's configuring these from inside `Datadog::Tracing.configure`.

Reply:
This 100% explodes, thank you!